pygsheets/worksheet.py

```diff
@@ -707,6 +707,22 @@ def create_named_range(self, name, start, end):
         self.client.sh_batch_update(self.spreadsheet.id, request, batch=self.spreadsheet.batch_mode)
         return DataRange(start, end, self, name)
 
+    def get_named_range(self, name):
+        """
+        get a named range given name
+
+        :param name: Name of the named range to be retrived, if omitted all ranges are retrived
+        :return: :class:`DataRange`
+
+        """
+        nrange = [x for x in self.spreadsheet.named_ranges if x.name == name and x.worksheet.id == self.id]
+        if len(nrange) == 0:
```

Comment:
Thanks, This would be more intuitive. But currently there is code duplication in `get_named_range` and `get_named_ranges`. Please remove the code in one function (preferably `get_named_ranges`)  and just reuse the 
other function.